tests/test_netsuite/test_views.py

```
@@ -236,3 +237,21 @@ def test_refresh_netsuite_dimensions(api_client, access_token, add_netsuite_cred
    response = api_client.post(url)
    assert response.status_code == 400
    assert response.data['message'] == 'NetSuite credentials not found in workspace'
+
+
+def test_netsuite_attributes_count_view(api_client, access_token, db):
```

Ruff (0.14.5)

Unused function argument: `db`

(ARG001)